README.md

```diff
@@ -1,2 +1,89 @@
-# DumE
-Open Projects 2021
+
+
+## DumE
+
+Open Projects 2021
+![DumE](https://github.com/rodion0917/Random/blob/main/Dum-E%20in%20Action%20-%20Green%20Object.png)
+***
+
+
+## Abstract
+
+Dum-E is a pick-up and place, robotic arm. Equipped with image processing and computer vision,it can 
+detect and classify objects on the basis of their color or shape. Currently, it can operate in static
+environments, that is where obstacles and the object are not moving.
+***
+
+
+## Motivation
+
+The name Dum-E, itself is a hint of our source if inspiration for doing this project. After all,who 
+hasn't watched Iron Man. Dum-E first appeared in Iron Man I, though it keep messing things for Tony, 
+it had saved his life too. 
+Our inspiration was simply to make something cool and learn the basics of robotics, computer vision,
+image processing and the various platforms which can be used for designing and simulating our robotic arm. 
+***
+
+
+
+## Workflow
+![Workflow](https://github.com/rodion0917/Random/blob/main/Work-flow%20chart.png)
+![Picking up the object - Workflow](https://github.com/rodion0917/Random/blob/main/Picking%20up%20the%20object%20-%20Workflow.png)
+![Placing the object - Workflow](https://github.com/rodion0917/Random/blob/main/Placing%20the%20object%20-%20Workflow.png)
+***
+
+
+
+## Applications
+
+Dum-E is a perfect robotic arm for performing simple tasks in factory settings. It can also be used in household for any kind of pick up and place tasks.
+***
+
+
+## Limitations 
+
+While we did our best, given the time duration of project, there is much space for going plus ultra. 
+Presently Dum-E can't function in a dynamic environment. Dum-E also has only a modest reach, which restricts it to lift only small, light objects.
+***
+
+
+## Future Improvements
+
+In future, we would work on making Dum-E function in a dynamic environment. Also, we try to enable Dum-E as a whole and not just its end-effector to move in dynamic environment. 
+***
+
+
+
+## Team Members 
+
+Abhay Pratap Singh<br/>
+Ayush Mishra<br/>
+Shivshankar Shukla<br/>
+Siddhant Shekhar<br/>
+***
+
+
+
+## Mentors
+
+Rishika Chandra
+***
+
+
+## References
+
+https://in.mathworks.com/help/physmod/sm/cad-import.html?s_tid=CRUX_topnav
+
+https://in.mathworks.com/help/physmod/sm/ug/import-a-cad-assembly-model.html
+
+https://in.mathworks.com/help/reinforcement-learning/ug/create-predefined-simulink-environments.html
+
+https://in.mathworks.com/help/robotics/ref/inversekinematics-system-object.html
+
+https://in.mathworks.com/help/robotics/ug/trace-end-effector-ik-simulink.html?searchHighlight=define%20the%20trajectory&s_tid=srchtitle
+
+https://in.mathworks.com/help/robotics/ug/2d-inverse-kinematics-example.html
+
+https://in.mathworks.com/help/robotics/ug/build-a-robot-step-by-step.html
+
+https://in.mathworks.com/help/robotics/ug/pick-and-place-workflow-using-stateflow.html

```

Comment:
Hyperlink the references properly. Mention the title for each reference and just include the URL in the hyperlink. Refer this README for more info: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md